### PR TITLE
New example and config changes

### PR DESCRIPTION
**Pull request by Stuey61296**
- Added another example for the react wrapper with a pyright language server
- Updated interfaces, types and configurations

### CODE REVIEW

**Comment by kaisalmen** [comment]
@Stuey61296 what about making the monaco-languageclient-examples package publicly available on npm. Then we could just take the server from there. This would also be helpful with the json languageserver. We have just keep the start scripts here and use the dependencies from node_modules. I will test this.

**Comment by Stuey61296** [comment]
> @Stuey61296 what about making the monaco-languageclient-examples package publicly available on npm. Then we could just take the server from there. This would also be helpful with the json languageserver. We have just keep the start scripts here and use the dependencies from node_modules. I will test this.

I think this a good idea, will definitely make it easier to maintain going forward. Do you need me to look into this or will you do it as part of the next release you do?

**Comment by kaisalmen** [comment]
> Do you need me to look into this or will you do it as part of the next release you do?

I will test this (release new packages) with the json example and if that works you could rebase this PR and do the same with the python server.

**Comment by kaisalmen** [comment]
@Stuey61296 Done. The server integrations for python and json LSs are already available on main. Please rebase the PR.

**Comment by kaisalmen** [comment]
I performed the rebase here already: https://github.com/TypeFox/monaco-components/tree/react-python

You can reset your branch to there if you like. The changes look good already and it works as expected.

File: packages/monaco-editor-wrapper/src/wrapper.ts [hunk 1]
```diff
@@ -58,8 +56,18 @@ export class MonacoEditorLanguageClientWrapper {
         this.serviceConfig.configureConfigurationService = this.serviceConfig.configureConfigurationService ?? {
             defaultWorkspaceUri: '/tmp/'
         };
+        if (wasVscodeApiInitialized()){
```
Comment (kaisalmen) [inline packages/monaco-editor-wrapper/src/wrapper.ts:59]: This is better then before. Thanks.

**Review by kaisalmen — COMMENTED** [review]
All the changes you made are fine and the corrections are great. Thank you.

In the rebase I did all your example and monaco-editor-wrapper changes are kept. Only the python server is now launched from the new `monaco-languageclient-examples`.

**Comment by kaisalmen** [comment]
I now merged the rebased version as there were no changes required from you side. Your name is on the changes! 🎉 Thank you. I only removed what I wrote above. I will release the new major versions later today.

**Comment by kaisalmen** [comment]
@Stuey61296 versions 3.0.0 and 2.0.0 are now available:
https://www.npmjs.com/package/monaco-editor-wrapper/v/3.0.0
https://www.npmjs.com/package/@typefox/monaco-editor-react/v/2.0.0

**Pull request closed without merging**